Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /nodeApp
 ADD . /nodeApp
 RUN ./make_version.sh
 
+# why is this not "npm install --production"? I'd like to exchange it with "npm prune --production"
```

Comment:
the reason we have two lines relates to the `napa` (https://github.com/shama/napa) dev module that imports the line-height plugin as a node module (it is not). That generally runs correctly with just `npm install`, but not when running as root (as is the case in deployment) because of https://github.com/npm/npm/issues/5637. Running `npm run install` overcomes that.

With your new approach I would guess that you could replace it with `npm prune --production` but still keep the `npm run install` command. That only runs `napa` and fetches the line-height plugin, which is a runtime-dependency.